libs/render_delegate/mesh.cpp

```diff
@@ -235,15 +234,13 @@ void HdArnoldMesh::Sync(
         }
     }
 
-    bool isLeftHanded = false;
+    // We have to flip the orientation if it's left handed.
+    const auto topology = GetMeshTopology(sceneDelegate);
+    bool isLeftHanded = topology.GetOrientation() == PxOsdOpenSubdivTokens->leftHanded;
```

Comment:
I'm a bit worried of moving the GetMeshTopology outside of the "if (dirtyTopology)" as this function seems to have a cost, and we can be called multiple times in Sync even when there's nothing to change.

I think we should either store a bool member _isLeftHanded in the class, of call GetMeshTopology again inside of "if (dirtyPrimvars)", of create an empty HdMeshTopology and set it only "if (dirtyPrimvars || dirtyTopology)" so that the topology can be used inside of both tests, but the more we can remove from outside of the dirtiness tests, the better